kent_a06b.js

```diff
@@ -0,0 +1,73 @@
+/*
+Tucker Kent
+kent_a06b.js
+19WI_INFO 2124_WW - Online - Javascript I
+Thoendel
+24 January 2020
+*/
+
+const standardInput = process.stdin;
+standardInput.setEncoding('utf-8');
+
+const story = 
+    `You're walking along Dodge Street minding your own business
+and your suddenly run into the Wizard of Benson standing
+in on the corner of 50th and Dodge.
+
+Do you (C)ontinue walking? (S)top and ask for some magical help?
+(T)rip and fall into a pot hole?
+`;
+
+clearScreen();
+console.log(story);
+
+standardInput.on('data', function(text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    /* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+if(text === "c" || text === "C"){ // OR logical operator used to process either case of c
+    clearScreen(); // calling clearScreen method -- clearing screen prior to response
+    console.log(`You non-challantly walk past the Wizard of Benson with the smug 
+confidence of a person who don't need no stinkin' Wizard, lest the 
+Wizard of Benson.`); //logging response of "c/C" to console
+    console.log("\n");
+    endGame(); // calling endGame method to display game over
```

Comment:
in each of your if/else sections you clear the screen, log some stuff and then print a blank line and endgame(). Is there a way you could combine some of this to be more efficient? May write your template to a variable and then clear, output, end after the if/else?